helper.fish

```diff
@@ -103,9 +111,13 @@ else ; set -gx VERBOSEBUILD $VERBOSEBUILD ; end
 function keepBuild ; set -gx NO_RM_BUILD 1 ; end
 function clearBuild ; set -gx NO_RM_BUILD ; end
 
+function compiler
```

Comment:
Can we add a test here if the input compiler version is valid?
Right now we only support:
`6.4.0`
`7.3.0`
`8.2.0`
with the build images.
Also i think it should be moved to `helper.linux.fish` as it does not work on Mac.

Reply:
Done